fr.txt

```diff
@@ -0,0 +1,405 @@
+AccountLinkRulesAccept = "Accepter et continuer"
+
+; Privacy Change Email
+; https://accounts.klei.com/account/email/change
+
+ChangeEmail = "Changer mon courriel"
+ChangeEmailDescription = "C'est l'adresse courriel qui nous utiliserons pour vous envoyer des nouveautés concernant votre compte ou vos jeux."
+EmailHasBeenChanged = "Votre courriel est en cours de modification. Nous vous avons envoyé un courriel avec un lien de vérification pour autoriser le changement."
+
+; Privacy Hide Inventory
+; https://accounts.klei.com/account/settings
+
+Inventory = "Inventaire"
+HideSteamInventory = "Masquer mon inventaire Steam"
+HideSteamInventoryDescription = "Cochez cette option pour masquer tous les objets associés à vos jeux Klei sur Steam. Vous possédez toujours vos objets et vous pouvez les voir dans le jeu. Si vous voulez rendre l'inventaire visible de nouveau, décochez l'option et sauvegardez."
+InventoryHasBeenHidden = "L'inventaire est masqué."
+
+; Marketing Options
+; https://accounts.klei.com/account/privacy
+
+MarketingOptions = "Options de marketing "
+MarketingOptionsDescription = ""
+MarketingOptionsUnsubscribe = "Je ne souhaite pas recevoir de courriel de marketing"
+MarketingOptionsSubscribe = "Je souhaite recevoir des courriels de marketing (nouvelles, jeux, promotions, etc)"
+MarketingOptionsDetails = "Pour des informations plus détaillées sur chaque jeu, merci de cocher les options suivantes:"
+MarketingOptionsPending = "Nous vous avons envoyé un courriel pour confirmer votre inscription à la newsletter."
+
+; GDPR Account Deletion
+; https://accounts.klei.com/account/limbo
+
+DeleteAccount = "Supprimer mon compte"
+DeleteAccountDescription = "Une fois votre compte supprimé, il n'y a pas de retour en arrière. Êtes-vous certain ?"
+DeleteAccountAreYouSure = "Êtes-vous absolument sûr ?"
+DeleteAccountWarning = "Des conséquences inattendues et désagréables vont arriver si vous ne lisez pas ceci !"
+DeleteAccountExplanation = "À votre demande, votre compte Klei et toutes vos informations personnelles reliés à votre compte (excluant la facturation et les abus) sont soumis pour suppression définitive par notre système.</p><p><b>Attention: vous ne pourrez pas annuler cette action, toutes vos données seront totalement et irréversiblement effacés. </b></p><p><strong>Cela signifie que vous ne pourrez plus jouer aux jeux nécessitant un Compte Klei (Don’t Starve Together et Hot Lava); le compte {{.PlatformName}} sera banni définitivement; et toutes les données seront détruites à jamais, incluant la progression, les objets et les récompenses.</strong>"

```

Comment:
Not related to translation, english text is misleading:
We don't ban the platform account. 
The user can eventually recreate a Klei account with his platform account

Reply:
same as English.